.ci/run.sh

```diff
@@ -24,6 +24,12 @@ x-pack-ciGroup*)
   export CI_GROUP="${JOB##x-pack-ciGroup}"
   ./test/scripts/jenkins_xpack_ci_group.sh
   ;;
+firefox-intake*)
+  ./test/scripts/jenkins_ci_group.sh
```

Comment:
I'd prefer if we just created new scripts for functional smoke tests, rather than wrapping so much of the logic in this script in if statements.